css/intro.css

```diff
@@ -198,12 +225,29 @@
 }
 @media (min-width: 1000px){
 	.header__contacts {
-		margin: 16px 0px 0px 86px;
+		margin: 22px 0px 0px 86px;
 	}
 }
 @media (min-width: 1920px){
 	.header__contacts {
-		margin: 16px 0px 0px 83px;
+		margin: 22px 0px 0px 83px;
+	}
+}
+.header__contacts svg{
```

Comment:
set classes for tags, and try not to use tags in selectors without a special reason

check all the such places

Reply:
Done.